### PR TITLE
About page updates

### PR DESCRIPTION
**Pull request by jospoortvliet**
Some updates to the about page, creating a first draft of a team page and contributors page. WIP!

* [x] basic layout for team and contributors page
* [x] add team images from @karlitschek min 500x500 (1000x1000 ideal?)
* [x] import contributors in contribook @LukasReschke 
* [ ] add team descriptions WIP

### CODE REVIEW

**Comment by LukasReschke** [comment]
>  import contributors in contribook @LukasReschke

Script to do that has been added to `/var/www/html/contribook`, to execute just run `php import-github.php` and new users  that marked their contributions public will be imported. 

At the moment it imports:

1. Real name (if available)
2. Location (if available)
3. Bio (if available)
4. Link to website (if available)
5. Link to GitHub
6. Profile picture from GitHub

An example can be seen at https://nextcloud.com/user/?user=andyscherzinger. Note that the data is imported and updating it later is a manual action at the moment. Except of the profile picture, that one is loaded directly from GitHub. So you likely need to adjust the template rendering a bit:

```
if($_['picture_200']<>''){
        echo('<span class="bloguserpicture"><img src="'.CONTRIBOOK_PHOTO_URL.$_['picture_200'].'" border="1" /></span>');
} elseif($_['github']) {
        echo('<span class="bloguserpicture"><img src="https://avatars2.githubusercontent.com/u/'.$_['githubId'].'?v=3&s=200" border="1" /></span>');
}
```

File: page-contributors.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+if(CONTRIBOOK) {
+
+	require(dirname(__FILE__).'/../../../contribook/main/contribook/lib_contribook.php');
+	$users=CONTRIBOOK_USER::getusers();
+	echo('<div class="row">');
+	foreach($users as $user) {
+		$data = CONTRIBOOK_USER::getuser($user);
+		echo('<div class="col-md-2">');
+			if(empty($data['picture_50'])) { $data['picture_50'] = 'blogger_50.png'; }
+			echo('<div class="contribimg"><span class="contribook_microbloguserpicture"><a href="/user/?user='.$user.'"><img class="img-responsive img-circle" src="'.CONTRIBOOK_PHOTO_URL.$data['picture_50'].'" border="0" /></a></span></div>');
```
Comment (LukasReschke) [inline page-contributors.php]: Adjust as described at https://github.com/nextcloud/nextcloud.com/pull/96#issuecomment-238089380

**Comment by jospoortvliet** [comment]
Still have some team stuff to add so for now, only the about and community pages go live. The team page comes later!

**Comment by LukasReschke** [comment]
@jospoortvliet I fixes some syntax errors in your PHP code. Also I added a shuffling of the array so that the entries are randomly sorted.

You can see the result at https://nextcloud.com/contributors/, note that all 15 minutes the order is shuffled. (as the wordpress page is cached and the shuffling happens in the PHP it's only shuffled when the cache is invalidated)

**Comment by LukasReschke** [comment]
9054dedf61748a3fd467aeb0bdbc419d4b1f7fc9, 4dd197b524f66bf00642d0e749c8ec7ff99cbf3e, 359a44831adb25f6204082f35e6c92793837c85f

**Comment by jospoortvliet** [comment]
Thanks @LukasReschke awesome!!!

**Pull request merged**